maps/yw/cryogaia-02-mining.dmm

```diff
@@ -19429,8 +19431,9 @@
 "Vg" = (
 /obj/item/weapon/pickaxe,
 /obj/structure/table/steel,
-/obj/item/device/binoculars,
 /obj/machinery/atmospherics/unary/vent_pump/on,
+/obj/item/weapon/storage/belt/utility,
+/obj/item/weapon/storage/belt/utility,
```

Comment:
Reduce the number of belts mining starts with downstairs to one or replace the belt upstairs in surface mining area with binocs would have made more sense if they were up there to begin with anyways.